site/src/modules/workspaces/WorkspaceOutdatedTooltip/WorkspaceOutdatedTooltip.tsx

```diff
@@ -17,102 +20,99 @@ import { usePopover } from "components/deprecated/Popover/Popover";
 import { linkToTemplate, useLinks } from "modules/navigation";
 import type { FC } from "react";
 import { useQuery } from "react-query";
-
-const Language = {
```

Comment:
I kinda like the pattern of putting all the language in a constant but it truly doesn't matter haha.